unit_tests/test_dbprocessing.py

```diff
@@ -84,6 +100,7 @@ def checkCommandLines(self, fid, expected):
 
     def testSimple(self):
         """Single daily file making another single daily file"""
+
```

Comment:
Any reason for adding this blank line here and elsewhere? PEP8 and PEP257 don't say to put a blank line, and the examples in 257 imply not.

Reply:
Nope, just a code convention you have and we don't.   We can remove the blank line after the docstring.

Reply:
Okay, will lump that in with other whitespace noodling.